### PR TITLE
Update `m1_setup.md`

### PR DESCRIPTION
**Pull request by tapabratadey**
Resolving changes requested in #374 on how to generate a local keypair. Apologies for opening another PR.

### CODE REVIEW

File: Solana_And_Web3/en/Section_2/Resources/m1_setup.md [hunk 1]
```diff
@@ -193,18 +193,28 @@ The solution right now is to have Anchor run with Solana's validator instead. Pr
...
 Okay, back to it! Let's open up a new terminal window and run:
 
-
 ```bash
 solana-test-validator --no-bpf-jit
 ```
 
+### 🔑 Create a local keypair.
+
+In order to talk to our Solana programs, let us generate a new keypair! We can run this command in our terminal:
```
Comment (adilanchian) [inline Solana_And_Web3/en/Section_2/Resources/m1_setup.md:202]: Can you change this to - 

In order to talk to our Solana programs, let's generate a new keypair! We can run this command in our terminal:

**Review by adilanchian — CHANGES_REQUESTED** [review]
Hey - 

Thanks so much for this! Just one more change hehe. Sorry! 

Also - you don't need to create PRs here, you can just update this branch with your changes :)

**Review by tapabratadey — COMMENTED** [review]
Thanks again!! Next time will make sure not to open a PR on a change request :p

File: Solana_And_Web3/en/Section_2/Resources/m1_setup.md [hunk 2]
```diff
@@ -193,18 +193,28 @@ The solution right now is to have Anchor run with Solana's validator instead. Pr
...
 Okay, back to it! Let's open up a new terminal window and run:
 
-
 ```bash
 solana-test-validator --no-bpf-jit
 ```
 
+### 🔑 Create a local keypair.
+
+In order to talk to our Solana programs, let's generate a new keypair! We can run this command in our terminal:
```
Comment (adilanchian) [inline Solana_And_Web3/en/Section_2/Resources/m1_setup.md:202]: I think we need just a tad bit more explanation here. Some like - 

In order for us to talk to our Solana programs we need to generate a keypair. Really all you need to know about this is it allows us to sign for transactions in Solana! Still curious? [Take a look at this page](https://solana-labs.github.io/solana-web3.js/classes/Keypair.html) for more information 

Let me know what you think!

**Review by adilanchian — CHANGES_REQUESTED** [review]
Sorry - just one more change here!!

File: Solana_And_Web3/en/Section_2/Resources/m1_setup.md (hunk 2 above)
Comment (tapabratadey) [reply Solana_And_Web3/en/Section_2/Resources/m1_setup.md]: Made the changes! Super nice documentation. Thanks for sharing!

**Review by tapabratadey — COMMENTED** [review]
Added description to  `Create a local keypair`. Thanks again!

**Review by adilanchian — APPROVED** [review]

**Comment by adilanchian** [comment]
@tapabratadey LOOKING GOOD!

Thanks so much for taking the time to add this :). It is indeed epic 🤘

**Comment by adilanchian** [comment]
@all-contributors please add @tapabratadey for doc

**Pull request merged**